Add search terms to arg dataframe commands

HannahZhuSWE opened the pull request:
# Description

Added a couple of search terms for the following commands https://github.com/nushell/nushell/issues/5093

Command | Search terms added
-- | --
arg-sort | argsort, order, arrange
arg-true | argtrue, truth, boolean-true
arg-unique | argunique, distinct, noduplicate, unrepeated
arg-max | argmax, maximum, most, largest, greatest
arg-min | argmin, minimum, least, smallest, lowest


# Tests


Make sure you've run and fixed any issues with these commands:

- [x] `cargo fmt --all -- --check` to check standard code formatting (`cargo fmt --all` applies these changes)
- [x] `cargo clippy --workspace --features=extra -- -D warnings -D clippy::unwrap_used -A clippy::needless_collect` to check that you're using the standard code style
- [x] `cargo test --workspace --features=extra` to check that all the tests pass

fdncred commented:
some of these like `sort` and `max` aren't necessary. maybe test them out with `help -f term` to see what is already highlighted in the hits.

HannahZhuSWE commented:
Sorry about that, I ran the search terms with `help -f term` and removed the duplicates (and updated the table to have just the remaining search terms).

fdncred commented:
no worries, thanks so much for helping out.
